llvm/test/Transforms/InstCombine/icmp-binop.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+; RUN: opt < %s -passes="instcombine<no-verify-fixpoint>" -S | FileCheck %s
```

Comment:
Why is this needed?

Reply:
Got `LLVM ERROR: Instruction Combining on icmp_eq_or_of_selects_with_scalar_cond_constant_vectorized did not reach a fixpoint after 1 iterations.` after running the test. Did I miss something in the code or is this an orthogonal issue?